api/components/examples/index.yaml

```diff
@@ -1,16 +1,27 @@
+EXTRA:
```

Comment:
Please remove `Extra` folder from examples. Please keep example in intra_city & metro use-case only.